pepper-apis/src/main/resources/db-changes/schema/DDP-4134-change-unique-keys-in-client-table.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sretsia" id="20200323-change-unique-keys-in-client-table">
+        <dropUniqueConstraint tableName="client" uniqueColumns="client_name" constraintName="client_name"/>
```

Comment:
Maybe I'm opening things up too much, but I wonder if we should just remove `client_name` from the database.  It's entirely redundant, since the client already has a name stored in auth0.  Doesn't seem like it's used very much.

Reply:
Done, removed `client.client_name` and changed the code using it.